react/react1/week2/new-todo-app/src/TodoList.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react';
+import './App.css';
+import TodoArray from './TodoArray';
+
+const todo = TodoArray();
+
+const randomText = [
+    'Brush teeth morning',
+    'Brush teeth afternoon',
+    'Brush teeth evening',
+    'Brush teeth night',
+];
+
+export function TodoList() {
```

Comment:
Try and see if you can separete into components. The TodoItem component goes into its own component. It has an Input component and a TodoDelete component